pallets/funding/src/functions.rs

```diff
@@ -1405,9 +1425,15 @@ impl<T: Config> Pallet<T> {
 		let now = <frame_system::Pallet<T>>::block_number();
 		let ct_usd_price =
 			project_details.weighted_average_price.ok_or(Error::<T>::ProjectError(ProjectErrorReason::WapNotSet))?;
-		let plmc_usd_price = T::PriceProvider::get_price(PLMC_FOREIGN_ID).ok_or(Error::<T>::PriceNotFound)?;
+		let plmc_usd_price = T::PriceProvider::get_decimals_aware_price(PLMC_FOREIGN_ID, USD_DECIMALS, PLMC_DECIMALS)
+			.ok_or(Error::<T>::PriceNotFound)?;
+
+		let funding_asset_id = funding_asset.to_assethub_id();
+		let funding_asset_decimals = T::FundingCurrency::decimals(funding_asset_id);
 		let funding_asset_usd_price =
-			T::PriceProvider::get_price(funding_asset.to_assethub_id()).ok_or(Error::<T>::PriceNotFound)?;
+			T::PriceProvider::get_decimals_aware_price(funding_asset_id, USD_DECIMALS, funding_asset_decimals)
+				.ok_or(Error::<T>::PriceNotFound)?;
+
```

Comment:
Could choose to add these three steps in a specific funding pallet function as we will be using it often:
getting asset id,
getting decimals
getting decimal aware price.

Reply:
We only use it bid(), and contribute(). I prefer to keep it verbose for now and abstract it if we keep running into it later. I'll keep this in mind